net/datacenter-node-install/setup-dc-node-2.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+if [[ "$EUID" -ne 0 ]]; then
+  echo "Please run as root"
+  exit 1
+fi
+
+set -xe
+
+./setup-cuda.sh
```

Comment:
Make path explicit

Reply:
At the file header, add an explicit path to the running script with something like:
`here=$(dirname "$0")`
Then later you can run:
`"$here"/setup-cuda.sh` or whatever, so you can call these scripts regardless of what your shell's PWD is.